### PR TITLE
test: extend beaker tests

### PR DESCRIPTION
**Pull request by b3lix**
test(Beaker): add check for additional keys
    
    Add check for additional keys [tasks, redention_tag, product]
    to beaker transformer tests.
    Resolve: https://github.com/neoave/mrack/issues/186

refactor(Beaker): transformer test to use dictionary
    
    Refactor beaker transformer test to use dictionary instead of multiple
    parameters in function call. This increase readability of test.

Resolve: https://github.com/neoave/mrack/issues/186

### CODE REVIEW

File: tests/unit/test_beaker_transformer.py [hunk 1]
```diff
@@ -114,67 +117,97 @@ async def create_transformer(self, legacy=False):
...
+                    "variant": "BaseOS",
+                    "ks_meta": "PROV_CONF_RHEL86_KS_META",
+                    "ks_append": [
+                        "%post\ncat /etc/redhat-release\nwget redhat.com\n%end"
+                    ],
+                    "whiteboard": default_whiteboard,
+                    "priority": default_prio,
+                    "tasks": default_tasks,
+                    "retention_tag": default_retention_tag,
+                    "product": default_product,
```
Comment (Tiboris) [inline tests/unit/test_beaker_transformer.py:181]: Do you think you can add some other than default values to one or two hosts ? 
I mean:
```
retention_tag
product
```
there I can see only defaults.

**Review by Tiboris — CHANGES_REQUESTED** [review]
Hi @b3lix !

Thanks for the patch it is very nice to see new contributor.
Also adding tests is much appreciated. Please find inline comment I would like you to try and update slightly the test.

File: tests/unit/test_beaker_transformer.py (hunk 1 above)
Comment (Tiboris) [reply tests/unit/test_beaker_transformer.py:240]: Thanks for the additions. Looks fine now.

**Comment by b3lix** [comment]
Thank you for your help @Tiboris. Glad to contribute!

**Pull request merged**